changelogs/enhancement-8282-enable-typedefs-woocommerce-components

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: Enhancement
+
+Added Typescript type declarations to build for @woocommerce/components #8282
```

Comment:
Could we move the changelog entry to `packages/components/CHANGELOG.md`? 

For package changes, we usually just put an entry into the package’s own changelog only as it's not a direct change to Woo Admin.

Reply:
Oh yea I keep forgetting, thanks!